traefik-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,57 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'traefik',
+        rules: [
+          // TraefikConfigReloadFailuresIncreasing
+          {
+            alert: 'TraefikConfigReloadFailuresIncreasing',
+            expr: |||
+              sum by (%(groupLabels)s) (rate(traefik_config_reloads_failure_total{%(filteringSelector)s}[5m])) > 0
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'critical',
+            } + std.get($._config, 'alertLabels', {}),
+            annotations: {
+              description: 'Traefik is failing to reload its config',
+            } + std.get($._config, 'alertAnnotations', {}),
+          },
```

Comment:
forgot to mention that it should be summary annotation in addition to description:  see how they should be used:
https://monitoring.mixins.dev/#guidelines-for-alert-names-labels-and-annotations

Reply:
good example! haven't used Grafana Alerts too much yet. added summary in c3ebbc91b11ce40af82042c41116bf2246eb7112